### ISSUE TITLE
Can't get high smatch scores

### ISSUE THREAD

**Issue by mahnerak** [nguyenlab/ConvAMR#2]
When we train ConvAMR on AMR 2.0 dataset with default settings, then generate AMRs, then calculate SMATCH score using this code: [https://github.com/snowblink14/smatch](https://github.com/snowblink14/smatch), we get the following scores on the test set:
```
Precision: 0.21176
Recall: 0.28149
F-score: 0.24170
```
The lowest value for the loss function we get is 3.34986 and the best perplexity is 10.21.

AMR2.0 is a superset of AMR1.0, so the results should not be very different. Can you please explain how you got 70+ SMATCH scores in your paper?
